### PR TITLE
updating the readme, installation doc, and csv to have more consistency

### PR DESCRIPTION
**Pull request by acornett21**
- Pointing our readme to the certification release readme 
   - For optional resources
   - For running the pipeline
- Updating our CSV to be more human readable
   - This matches the readme minus the *Install* step  

Signed-off-by: Adam D. Cornett <adc@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,233 +38,13 @@ oc create secret generic github-api-token --from-literal GITHUB_TOKEN=<github to
...
-  --workspace name=pyxis-api-key,secret=pyxis-api-secret \
-  --showlog
-```
-
-## <a id="submit-result-registy-and-pinning"></a>Submit results with Image Digest Pinning and a private container registry
-* Execute the [Configuration Steps for Submitting Results](#step7)
-* Execute the [Configuration Steps for Digest Pinning](#digest-pinning-config)
-* Execute the [Configuration Steps for Private Registries](#private-registry)
+# Installation Steps
+Since this operator isn't in OperatorHub, the process to get it into a cluster is manaul at this point.
```
Comment (dcurran90) [inline README.md:45]: manual*

File: bundle/manifests/operator-certification-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -37,25 +37,63 @@ spec:
...
+    - [Install](https://docs.openshift.com/container-platform/4.8/cli_reference/openshift_cli/getting-started-cli.html#installing-openshift-cli) oc, the OpenShift CLI tool (tested with version 4.7.13)
+    - [Install](https://tekton.dev/docs/cli/) tkn, the Tekton CLI tool (tested with version 0.19.1)
+    - [Install](https://git-scm.com/downloads) git, the Git CLI tool (tested with 2.32.0)
+    - The certification pipeline expects you to have the source files of your Operator bundle
+
+
+    **Pre - Installation**
+
+
+    The below steps exist for simplicity and to make the instillation clearer. The operator watches all namespaces, so if secrets/configs/etc already exist in another namespace, feel free to use the existing namespace when following the operator installation steps.
```
Comment (dcurran90) [inline bundle/manifests/operator-certification-operator.clusterserviceversion.yaml:57]: s/instillation/installation/

File: README.md [hunk 3]
```diff
@@ -35,233 +38,13 @@ oc create secret generic github-api-token --from-literal GITHUB_TOKEN=<github to
...
-# Installation Steps
-Since this operator isn't in OperatorHub, the process to get it into a cluster is manaul at this point. 
-Please follow the below steps to get the operator into your cluster. Follow the steps from [here](docs/INSTALLATION.md)
-
-# <a id="execute-pipeline"></a>Execute the Pipeline (Development Iterations)
-There are multiple ways to execute the Pipeline.  Below are several examples but parameters and workspaces can be removed or added per your requirements.
-
-## Clone the pipelines repository locally so you have the necessary input files to pass to tkn commands.
-
-* `git clone https://github.com/redhat-openshift-ecosystem/operator-pipelines`
```
[the remark this answers is not in the document]
Comment (acornett21) [reply README.md:47]: Added this into the readme and csv but used the `operator-pipelines` config instead of the above.

**Review by bcrochet — APPROVED** [review]
/lgtm

**Review by jomkz — APPROVED** [review]

**Pull request merged**